### PR TITLE
update release tarball structure

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

**Review by cmars — APPROVED** [review]

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2194/

File: Makefile [hunk 1]
```diff
@@ -130,11 +130,10 @@ gopkg:
...
 # Build a release tarball
 charmstore-$(VERSION).tar.xz: $(VERSIONDEPS)
-	mkdir -p charmstore-$(VERSION)/bin
-	GOBIN=$(CURDIR)/charmstore-$(VERSION)/bin go install $(INSTALL_FLAGS) -v $(PROJECT)/...
-	mv charmstore-$(VERSION)/bin/charmd charmstore-$(VERSION)/bin/charmstore
-	tar cv charmstore-$(VERSION) | xz > $@
-	-rm -r charmstore-$(VERSION)
+	mkdir -p charmstore-release/bin
+	GOBIN=$(CURDIR)/charmstore-release/bin go install $(INSTALL_FLAGS) -v $(PROJECT)/...
+	tar cv -C charmstore-release . | xz > $@
```
Comment (rogpeppe) [inline Makefile:135]: Not sure I'd bother with the "v". Then again I wouldn't bother with the "-v" on the go install either. Just taste I guess. *shrug*

**Review by rogpeppe — APPROVED** [review]
LGTM with one trivial thought, thanks.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2195/

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-merge

**Pull request merged**